### ISSUE TITLE
Avoid GitHub API failures (Release Notes)

### ISSUE THREAD

**Issue by pgorod** [salesagility/SuiteDocs#483]
#### Page with Issue

Our Release Notes pages use the GItHub API to fetch members' avatars. Currently, API failures might cause problems when building the site. And API failures are not _that_ uncomon - not full GitHub outages, but a much simpler thing: exhausting API limits due to too many requests when building the site.

#### Underlying issue

This is related to a Hugo known issue, https://github.com/gohugoio/hugo/issues/6954

After a long time stalled, recently there was a fix for this, check it out.

#### What needs to be done here on SuiteDocs

- wait for the fix to be released and work to upgrade our Hugo to that release
- understand how it works and adjust our API code to fail gracefully (build the site without interruption, but use a placeholder image if it can't download the avatar).

This is the file that needs to get the improved error handling:

https://github.com/salesagility/SuiteDocs/blob/master/layouts/shortcodes/ghcontributors.html

**Comment by samus-aran** [comment]
Yup! This is what I now face.... Either jump ahead and provide the fall back now and work on the upgrade. I'm not sure if they can be done separately but an upgrade is always good.

**Comment by serhiisamko091184** [comment]
Hello @pgorod, 

I'd like to check if this issue is still valid? 

Thanks in advance! 

Regards,
Serhii

**Comment by pgorod** [comment]
I guess it is. In general, the whole Docs stack should get some love (and upgrades). That would be Hugo, the Learn theme, asciidoc.
